src/e2e-test/resources/pluginParameters.properties

```diff
@@ -1,6 +1,7 @@
 projectId=cdf-athena
 datasetprojectId=cdf-athena
 dataset=bq_automation
+wrongSourcePath=gs://abcdefgh
```

Comment:
would be better to use a random string (of same length or more)

Reply:
Done. Using the Value of 'wrongSourcePath' with the same bucket length.